keep-only-warm-data/docker/kuzzle/custom_start.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+npm --prefix /scripts install /scripts
```

Comment:
add the `--unsafe` option to allow npm to operate in images with elevated privileges.

Reply:
Why ? It already work like this

Reply:
It is, until you add a dependency that needs to be compiled ;-)